src/cascadia/LocalTests_SettingsModel/SerializationTests.cpp

```diff
@@ -174,7 +177,8 @@ namespace SettingsModelLocalTests
 
         const std::string smallProfileString{ R"(
             {
-                "name": "Custom Profile"
+                "name": "Custom Profile",
+                "font": {}

```

Comment:
not a big fan of `"font": {}` getting added into these. Could we check if the font config is empty, and if it is, don't write it out?

Reply:
Good call, fixed!